e2e/content-services/upload/uploader-component.e2e.ts

```diff
@@ -93,8 +85,7 @@ describe('Upload component', () => {
             for (const node of nodeList) {
                 try {
                     await uploadActions.deleteFileOrFolder(node);
-                } catch (error) {
-                }
+                } catch (error) {}
```

Comment:
Is try catch here needed?

Reply:
the e2e probably handles missing nodes and does not crash on deletion